app/controllers/Application.scala

```diff
@@ -32,6 +32,10 @@ object Application extends Controller {
   ) (Votes.partialApply _)
 
 
+  implicit val openChanceWrites = new Writes[OpenChance] {
```

Comment:
Why implicit? I'm not sure it's necessary

Reply:
It looks like an usual way to deal with it